### PR TITLE
fix:Multilingual preflabels

### PR DESCRIPTION
**Pull request by alexskr**
see: https://github.com/ncbo/ontologies_linked_data/pull/220

### CODE REVIEW

File: test/controllers/test_classes_controller.rb [hunk 1]
```diff
@@ -208,7 +208,7 @@ def test_roots_for_cls
     roots = MultiJson.load(last_response.body)
     assert_equal 9, roots.length
     roots.each do |r|
-      assert_instance_of String, r["prefLabel"]
+      assert_instance_of String, r["prefLabel"], r["@id"]
```
Comment (syphax-bouazzouni) [inline test/controllers/test_classes_controller.rb:211]: why this changes?

Comment (mdorf) [reply test/controllers/test_classes_controller.rb:211]: when this assertion fails (for example if prefLabel = nil), it's informative to know the ID of the term that caused this failure
